libs/sdk-ui-ext/src/internal/interfaces/InsightView.ts

```diff
@@ -122,3 +121,10 @@ export interface IInsightViewProps extends Partial<IVisualizationCallbacks> {
      */
     TitleComponent?: React.ComponentType<IInsightTitleProps>;
 }
+
+/**
+ * @public
```

Comment:
We had to move it because we had problem with dependency ... discussed this with Dan

Reply:
Internal is isolated module so only chance to make it work si move this interfaces to Internal

Reply:
It is as Ivan says, they had to be moved here from the public part of the package to avoid cyclic/not wanted internal dependencies. The types themselves however need to stay `public`